documentation/source/user_section/tutorials/registration_to_template.rst

```diff
@@ -0,0 +1,375 @@
+.. _registration-to-template:
+
+Registration to template
+########################
+
+This tutorial demonstrates how to use SCT's command-line scripts to register an anatomical MRI scan to the PAM50 Template, then use the results of this registration to perform basic quantitative analysis. We recommend that you read through the :ref:`pam50` page before starting this tutorial, as it provides an overview of the template included with SCT, as well as context for why it is used.
+
+.. warning::
```

Comment:
warning a bit strong, don't you find? how about "tips" or "info"? or something lighter